clisops/core/subset.py

```diff
@@ -1043,18 +1040,9 @@ def subset_shape(
         try:
             shape_crs = CRS(poly.crs)
         except CRSError:
-            minbnd, _, maxbnd, _ = poly.total_bounds
-            # This is guessing that lons are wrapped around at 180+ but without much information, this might not be true
-            if minbnd >= -180 and maxbnd <= 180:
-                shape_crs = wgs84
-            elif minbnd >= 0 and minbnd <= 360:
-                shape_crs = wgs84_wrapped
-            else:
-                raise CRSError(
-                    "Shapefile CRS could not be determined and does not resemble WGS84."
-                )
```

Comment:
This logic is needed to know when a shape might be crossing the day-line. The `wgs84_wrapped` PROJ4 string is needed to ensure that the shape corresponds to gridded data that is starting from Greenwich.

Reply:
I guess we think it is more unlikely that the guess is wrong here compared to `raster_crs` ?

Reply:
We could add more conditionals here to figure out whether we might be looking at geographic data (like latitudes between -90 and 90?), but often times projected data will have very large values (pertaining to increments of feet or meters). It's rare that you would find projected shapes that have values this small (therefore, it's likely a geographic coordinate system).